src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -661,6 +662,8 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 						Tag t = af.getTag();
 
 						if (t != null) {
+							addMusicBrainzIDs(af, file, audio);
```

Comment:
Not sure if this applies here too, but did you see this commit I did? https://github.com/UniversalMediaServer/UniversalMediaServer/pull/2882/commits/a3844ee54e2d368e51d208a3355008f4e8927a45
I found that we were trying to parse tags for videos, which would always fail, so we should be careful to not do that here too

Reply:
The MBID attributes in line 665 will only be added, if we have an audio tag. So this section should not be applicable to video files ... Therefore I think it's ok.